entrypoint.sh

```diff
@@ -4,13 +4,23 @@ set -euo pipefail
 bento_path=$(which bento)
```

Comment:
At some point we should probably just make this a Python script

Reply:
Yep, that's my plan as well. Using click to allow configuration via environment variables for the whole thing would be really useful, too.